tools/browser-matcher/common-matchers.mjs

```diff
@@ -0,0 +1,5 @@
+import SpecMatcher from './spec-matcher'
+
+// Does the browser have a reliable 'unload' event callback?
+export const notInternetExplorer = new SpecMatcher()
+  .exclude('ie')
```

Comment:
Are we blending common matchers with bespoke ones in each test, or adding all matchers here?

Reply:
I suggest we create shared/common matchers in this file and import them into the tests. If a test needs a one-off matcher, it can be created in that test.